scripts/execute_script.sh

```diff
@@ -6,7 +6,10 @@
 
 set -e # stop the script if any subprocess fails
 
-etherlime compile --runs 999
+# rm -rf build
+# npm run compile:lib
+# npm run compile
+# npm run compile:legacy
```

Comment:
I'm for this change considering how long compilation takes currently however the obvious risk is that we execute scripts without compiling to latest version of the contracts. As a prevention, can we add some warning text in the comments header of scripts using `execute_script` to say `COMPILE CONTRACTS BEFORE RUNNING` or something similar?

Reply:
Agreed. Actually I've added a `--no-compile` flag in #156 (https://github.com/argentlabs/argent-contracts/pull/156/files#diff-0aa9ab67bc9291891ca0b035f24dce95R12) to address that. We can either wait for #156 to be merged or I can include that change in this PR.

Reply:
I don't mind either way.